pkg/nuctl/command/import.go

```diff
@@ -112,9 +113,41 @@ func (i *importCommandeer) importFunction(ctx context.Context, functionConfig *f
 			FunctionConfig: *functionConfig,
 		})
 
+	if err != nil && autofix {
+		typedErr := err.(*errors.Error)
+		return i.retryImportWithAutofix(ctx, functionConfig, typedErr)
+	}
+
 	return err
 }
 
+func (i *importCommandeer) retryImportWithAutofix(ctx context.Context, functionConfig *functionconfig.Config, err *errors.Error) error {
+	// TODO: add maxRetry value
+	i.rootCommandeer.loggerInstance.WarnWithCtx(ctx, "Function import was failed, it will be retried",
+		"function", functionConfig.Meta.Name,
+		"error", err.Error())
+
+	if strings.Contains(errors.GetErrorStackString(err, 10), "V3IO Stream trigger does not support autoscaling") {
+		i.rootCommandeer.loggerInstance.WarnWithCtx(ctx, "Setting maxReplicas to minReplicas for function",
+			"function", functionConfig.Meta.Name)
+		functionConfig.Spec.MaxReplicas = functionConfig.Spec.MinReplicas
+	}
```

Comment:
I had the same idea about including it in functionconfig, but I believe we would derive greater benefits from modifying our validation approach to introduce something like this

```
type functionValidator struct {
           validationError error
           validation func
           autofix      func
}
```
Since this fix is a high priority now, I'd either leave as is and leave the moving idea for the next PR or just move this function to functionconfig file as you suggested.